### ISSUE TITLE
Deploying updated docker file bug

### ISSUE THREAD

**Issue by omerb01** [pywren/pywren-ibm-cloud#133]
Found a bug with the new action cloning logic:
I had actions installed with my docker env on Cloud Functions and tried to deploy a new env with a new docker file by `./deploy_runtime create omerb01/pywren-omer-runtime:3.6`
after that, the current actions that installed in my space on Cloud Functions didn't update and I had to delete them manually from Cloud Functions and COS to reinstall them with the recent changes.

**Comment by JosepSampe** [comment]
@omerb01 Currently `./deploy_runtime` script updates runtimes one by one. I guess you have same docker image with different memory sizes in your CF, right? The `create` command will only update the action with 256Mb of memory, which is the default in the config file. To update the others runtimes you don't need to delete them from CF, you can run:

The `clone` command if your docker image has different packages to the previous version:
```
./deploy_runtime clone omerb01/pywren-omer-runtime:3.6 --memory 1024
./deploy_runtime clone omerb01/pywren-omer-runtime:3.6 --memory 512
etc..
```
The `update` command if your only want to update PyWren code:
```
./deploy_runtime update omerb01/pywren-omer-runtime:3.6 --memory 1024
./deploy_runtime update omerb01/pywren-omer-runtime:3.6 --memory 512
etc..
```

**Comment by omerb01** [comment]
@JosepSampe yes, I do have same docker images with different memory sizes but I think all this memory treat logic should be encapsulated from the user. the user / the developer shouldn't be aware of which actions were deployed and which wren't

**Comment by JosepSampe** [comment]
Fixed in #139
@omerb01 With this, if you modify Dockerfile, you can simply run to update all actions in your namespcea based on this docker image:
`./deploy_runtime build omerb01/pywren-omer-runtime:3.6`

If you only modified the source code, you can run to update all the actions based on this image:
`./deploy_runtime update omerb01/pywren-omer-runtime:3.6`
